src/main/java/uk/gov/digital/ho/hocs/audit/aws/listeners/AuditListener.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.digital.ho.hocs.audit.aws.listeners;
+
+import com.google.gson.Gson;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.cloud.aws.messaging.listener.SqsMessageDeletionPolicy;
+import org.springframework.cloud.aws.messaging.listener.annotation.SqsListener;
+import org.springframework.stereotype.Service;
+import uk.gov.digital.ho.hocs.audit.AuditDataService;
+import uk.gov.digital.ho.hocs.audit.auditdetails.dto.CreateAuditDto;
+
+@Service
+@Slf4j
+public class AuditListener {
```

Comment:
Much like we have the controllerAdvice, do we not want to use the @ MessageExceptionHandler() for here